infrastructure/aws-accounts/private/hernan.rajchert.password

```diff
@@ -0,0 +1 @@
+wcFMA9PsIunb9q6CARAA0VzuaHkPjcgzRhVBflr/0uyKSZyMJZ2xDZX4YoQC+Y7inqyC9r/5tc7D2rX6dgkRYUqOAgQM4En/p5kpN4BMdogjhGa2W9l9ods+7gbWpm2kAwrBSII0ihoX5dDaGgXDiLPxPc4JuHiPtvsmgQI/ebYi6Es4IVzXzUd+XRd1aHU+D4fSNP+SItI2MBM08pQ9KaEOTHgPjtZrKpQgOOQFYFPq9Ylm3HAxOIX6Uto1OzuNyHCpSZw+L9/0ygSQpk7y4frN2PJHwPngJkFu9d2U5/bOFhZh7PZ9JWhFOwlI+c6tY+Gas3pJpfBEQ07x63S5Z2hMzFbgzppalYEXx2KoZ3uL6fKyagCurM0szT6447RF1oOw5HSjhRIKCpZgjLodSUY8laOoI0Tbe+1lUuYIKIrKpYP9tpHbZdX+xBHzVoKXGCAaEUx4tA3x4acm8TCR5SIbhZf4W1Q5F3F33sYYQjwmtl7S9YgwJTZ7dz+sfIL5gGc0rwUPzN3gDAGbgiBAp2oudY3n1NvL6r9BZjPDmZOuaHJD6hoYrdAhZgcXX97IOPXe6KJDB8661KKm31WGO9cFmIz3si68IWN2n/RoNcoOSG8clqs0YqwmxqIXak7MoOmpPxZKQGUduMXuZ7Fk2J0cjK6H0a/ORbk8Ggo5ERbzvTjUP65M4IaPxv4KrMzS4AHkfIzLN2jnS5R3keEd5/0s2+HGjODS4DnheBDgdeJN5XtZ4FbkQGGM2VnX9n3Dwq83Z9INzeB84u71c/HgVeRALx2fHESjR9k19ZUsTPKN4iSjHunh0pIA
```

Comment:
Should these files be checked in?

Reply:
it's encrypted and makes the life of terraform easier so I don't think it does any harm

Reply:
Then should we be checking in similar files for the other users?

Reply:
if they were imported they don't have one.